### ISSUE TITLE
Name property is not provided for button present in 'Curl' and 'Response body' code.: A11y_VS Code extension for API Center_OpenAPIDocumentation_CodeExecute_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#113]
### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #ORCA; #Linux; #FTP; 

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4
Screen Reader: ORCA

### Repro Steps:

1. Start ORCA screen reader.
2. Create a API center in azure portal. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the api name which you have created and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7. Demo conference API screen will appear.
8. Tab till sessions dropdown button and press enter. Tab till try it out and press enter.
9. Tab till execute and press enter.
10. Navigate using TAB key to button present in 'Curl' and 'Response body' code and observe the issue.

### Actual Result:
Name property is not provided for button present in 'Curl' and 'Response body' code.

### Expected Result:
Appropriate name property should be provided for button present in 'Curl' and 'Response body' code.

### User Impact:
Screen reader users will not get the information about the control and it purpose.

### Attachment:

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222927

**Comment by Yash14j** [comment]
Github Tags:

#External;#External:SwaggerUI;#eDAD:3p:222942;#ThirdParty

**Comment by AnKushSingh05** [comment]
External bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222942

**Comment by AnKushSingh05** [comment]
External bug: https://github.com/swagger-api/swagger-ui/issues/10319
